### PR TITLE
#1 - Added Packages class for saving packages registry at key built b…

### PR DESCRIPTION
**Pull request by olegmoz**
…y package name

### CODE REVIEW

File: src/test/java/com/artpie/composer/Resource.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import com.google.common.io.ByteStreams;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Java resource read from class loader.
+ *
+ * @since 0.1
+ */
+final class Resource {
```
Comment (yegor256) [inline src/test/java/com/artpie/composer/Resource.java]: @olegmoz you can replace it with `ResourceOf` from Cactoos, which does exactly the same

File: src/main/java/com/artpie/composer/Packages.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        this.name = name;
+        this.content = content;
+    }
+
+    /**
+     * Saves packages registry binary content to storage.
+     *
+     * @param storage Storage to use for saving.
+     */
+    public void save(final BlockingStorage storage) {
```
Comment (yegor256) [inline src/main/java/com/artpie/composer/Packages.java]: @olegmoz I think we should accept `Storage` here and make it "blocking" inside

**Comment by olegmoz** [comment]
@yegor256 Thank you, both issues are fixed now. Please review

**Comment by olegmoz** [comment]
@yegor256 Fixed conflict with master branch, PR ready to be merged.

**Comment by yegor256** [comment]
@olegmoz thanks!

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#7` is not assigned, can't get performer

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**